flashpay/apps/account/views.py

```diff
@@ -175,3 +178,35 @@ def post(self, request: Request) -> Response:
             },
             status=status.HTTP_200_OK,
         )
+
+
+class APIKeyView(ListCreateAPIView):
+    def get_queryset(self) -> QuerySet:
+        return APIKey.objects.filter(account=self.request.user)  # type: ignore
```

Comment:
Why type ignore?

Reply:
request. user is expected to be `AbstractUser` while the account is `Account`. Unless we want to subclass.

Once again, if you have an alternative method, suggest